src/export.rs

```diff
@@ -22,31 +22,32 @@ use iris_lib::color::Color;
 /// ```
 ///
 pub fn write_html_out(image_file_path: &str, color_data: &[Color], out_file_path: &str) -> std::io::Result<()> {
-    let mut file = File::create(format!("{}.html", out_file_path))?;
-    file.write_all(b"<!DOCTYPE html><head>")?;
-    file.write_all(b"<meta content=\"width=device-width, initial-scale=1\" http-equiv=\"Content-Type\">")?;
-    file.write_all(b"<meta content=\"utf-8\" http-equiv=\"encoding\">")?;
-    file.write_all(b"<style> img {display: block; margin-left: auto; margin-right: auto; }</style>")?;
-    file.write_all(b"</head>")?;
-    file.write_all(b"<html><body>")?;
-    let image_element = format!("<img src=\"{}\" alt=\"Input image\" class=\"centered\">", image_file_path);
-    file.write_all(image_element.as_bytes())?;
-
-    file.write_all(b"</b>")?;
+    let mut html: String = String::new();
 
-    file.write_all(b"<div style=\"display: grid;align-items: center;justify-content: center;gap: 5px;width: 100%;padding-top: 10px;grid-auto-flow: column;\">")?;
+    html.push_str("<!DOCTYPE html><head>");
+    html.push_str("<meta content=\"width=device-width, initial-scale=1\" http-equiv=\"Content-Type\">");
+    html.push_str("<meta content=\"utf-8\" http-equiv=\"encoding\">");
+    html.push_str("<style> img {display: lock; margin-left: auto; margin-right: auto; }</style>");
```

Comment:
There is a small typo here. You meant `block`, which became `lock` here.
```suggestion
    html.push_str("<style> img {display: block; margin-left: auto; margin-right: auto; }</style>");
```